src/Php84/bootstrap.php

```diff
@@ -15,6 +15,11 @@
     return;
 }
 
+if (!defined('CURL_HTTP_VERSION_3')) {
+    define('CURL_HTTP_VERSION_3', 30);
+    define('CURL_HTTP_VERSION_3ONLY', 31);
+}
```

Comment:
In php/php-src#15350 the constants are only declared if the linked libcurl exposes them. Your patch however would register the constants unconditionally, even if ext-curl isn't loaded at all. cURL constants are often used for feature detection, which is why we should make sure they are available only in cases where PHP would also declare them.